src/lib/isContainerID.js

```diff
@@ -0,0 +1,28 @@
+import assertString from './util/assertString';
+
+// https://en.wikipedia.org/wiki/ISO_6346
+const isContainerIdReg = new RegExp('^[A-Z]{3}U[0-9]{7}$');
+const isDigit = new RegExp('^[0-9]{1}');
```

Comment:
I feel like this could be rewritten to `^[0-9]$`, since you are checking against each string character using its index, and that will *always* be just one character

Also maybe just a matter of taste in general, but this project seems to be generally prefering the use of regular expression literals. as opposed to a constructor function, like you used here.
So for the sake of style consistency, I would probably also suggest the following change:

`new RegExp('^[0-9]{1}')`-> const isDigit = `/^[0-9]$/`
(same applies to the isContainerIdReg regexp in line 4)

Reply:
Hi Panagiotis,

Thank you very much for pointing it out! I have updated the corresponding lines.